subprojects/emcee/worker/build.gradle.kts

```diff
@@ -12,11 +12,14 @@ dependencies {
     implementation(libs.okhttp)
     implementation(libs.okhttpLogging)
     implementation(libs.moshi)
+    implementation(libs.bundles.ktor)
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.2")
```

Comment:
Do you plan to move versions to toml file?

Reply:
There is an old version already. But it has compatibility problems with current Kotlin version. We can discuss it at the next sync.

Reply:
I think it's not a problem to have it in toml file